tests/common/snappi_tests/qos_fixtures.py

```diff
@@ -106,3 +111,55 @@ def lossy_prio_list(all_prio_list, lossless_prio_list):
     """
     result = [x for x in all_prio_list if x not in lossless_prio_list]
     return result
+
+
+def get_pfcwd_config(duthost):
+    config = get_running_config(duthost)
+    if "PFC_WD" in config.keys():
+        return config['PFC_WD']
+    else:
+        all_configs = []
+        output = duthost.shell("ip netns | awk '{print $1}'")['stdout']
+        all_asic_list = output.split("\n")
+        all_asic_list.append(None)
+        for space in all_asic_list:
+            config = get_running_config(duthost, space)
+            if "PFC_WD" in config.keys():
+                all_configs.append(config['PFC_WD'])
+            else:
+                all_configs.append({})
+        return all_configs
+
+
+def reapply_pfcwd(duthost, pfcwd_config):
```

Comment:
diffrence with existing function?

Reply:
@sdszhang , @amitpawar12 : Added a comment in the script.